lib/parser.ex

```diff
@@ -0,0 +1,50 @@
+defmodule DateTimeParser.Parser do
+  @moduledoc false
+  alias DateTimeParser.{Epoch, Serial}
+
+  @epoch_regex ~r|\A(?<seconds>\d{10,11})(?:\.(?<subseconds>\d{1,10}))?\z|
+  @serial_regex ~r|\A-?\d{1,5}(?:\.\d{1,10})?\z|
+  @default_parsers [:epoch, :serial, :tokenizer]
```

Comment:
When looking into if this could be pulled into hexdocs easily, I found [this post](https://til.hashrocket.com/posts/afkxgihz2v-accumulating-attributes-in-elixir), which seemed sort of interesting fo this use case. I'm not sure it's better, but figured I'd share.

Essentially, you could list each of the default parsers on their own line and accumulate them into a list at compile time.

In the example, they also define a function to return the module attribute, doing that could make doc stuff easier since [functions can be auto-linked](https://github.com/elixir-lang/ex_doc#auto-linking).